chcemvediet/apps/inforequests/admin.py

```diff
@@ -394,18 +410,62 @@ def delete_constraints(self, obj):
                 u'{} is the only action in the branch.'.format(admin_obj_format(obj))))
         return constraints
 
+    def can_snooze(self, obj):
```

Comment:
Premenoval by som to na `can_snooze_previous_action`, nech je jasne o co sa jedna.

Reply:
:ok: